src/stories/RegistrationSuccess.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import RegistrationSuccess from 'components/RegistrationSuccess';
+import { GlobalProvider } from 'context/global';
+export default {
+  title: 'Registration Success',
+  component: RegistrationSuccess,
+};
+
+export const Registration_Success = () => {
+  return (
+    <>
+      <div style={{ color: '#E5E5E5' }}>
+        <GlobalProvider>
```

Comment:
you can remove GlobalProvider since we are not using Context API in this component